src/main/java/com/google/cloud/hadoop/io/bigquery/hive/WrappedBigQueryAvroOutputFormat.java

```diff
@@ -27,40 +28,68 @@
 import org.slf4j.LoggerFactory;
 
 /*
- * Wrapper class for BigQuery writes.
+ * Class {@link WrappedBigQueryAvroOutputFormat} serializes hive records as Avro records and writes to file on GCS
  */
+
 public class WrappedBigQueryAvroOutputFormat extends AvroContainerOutputFormat {
-    private static final Logger LOG = LoggerFactory
-        .getLogger(WrappedBigQueryAvroOutputFormat.class);
-    public static final String TEMP_NAME = "_hadoop_temporary_";
+    private static final Logger LOG = LoggerFactory.getLogger(WrappedBigQueryAvroOutputFormat.class);
+    private static final String TEMP_NAME = "_hadoop_temporary_";
 
+    /**
+     *  Create Hive Record writer.
+     *
+     * @param jobConf Hadoop Job Configuration
+     * @param path  Hadoop File Path
+     * @param valueClass Class representing record (in this case AvroGenericRecordWritable)
+     * @param isCompressed Indicates whether records are compressed or not
+     * @param properties  Job Properties
+     * @param progressable Instance to represent the task progress
+     * @return  Instance of Hive Record Writer
+     * @throws IOException
+     */
     @Override
     public RecordWriter getHiveRecordWriter(JobConf jobConf, Path path,
         Class<? extends Writable> valueClass, boolean isCompressed,
         Properties properties, Progressable progressable) throws IOException {
 
         Path actual = new Path(getTempFilename(jobConf));
-        LOG.warn("Set temporary output file to {}", actual.getName());
+        LOG.info("Set temporary output file to {}", actual.getName());
 
-        return super.getHiveRecordWriter(
-                    jobConf, actual, valueClass, isCompressed, properties, progressable);
+        return super.getHiveRecordWriter(jobConf, actual, valueClass, isCompressed, properties, progressable);
```

Comment:
I believe this line is extra long. Refer to the Java style guide and use a linter to find out things like this.

Reply:
I installed Google style Guide plugin (https://github.com/google/google-java-format) and reformatted entire code.